### DISCUSSION TITLE
Orange Pi support

### DISCUSSION THREAD

**Discussion by rolandow** [mKeRix/room-assistant#1002] [Q&A]
Has anyone tried a Orange Pi yet? The raspberry pi zero w is out of stock in the stores I have access to for a while now. I think the Orange PI i96 could be a nice alternative, because it already has bluetooth on board. 

Any experiences with this type of board?

**Comment by odhiambo** [comment]
Is Orange Pi supported by Home Assistant? Does Orange Pi have Bluetooth support? If both answers are yes, then I don't see why RA should not work on it, given that you can install RA as an addon to HA:
https://peyanski.com/presence-detection-using-room-assistant-home-assistant-and-raspberry-pi/
